virttest/migration.py

```diff
@@ -132,6 +132,10 @@ def ping_vm(self, vm, params, uri=None, ping_count=10,
             func = exceptions.TestFail
             uri_backup = vm.connect_uri
             vm.connect_uri = uri
+            vm.cleanup_serial_console()
```

Comment:
Do line 135 and 136 are necessary?

Reply:
They are necessary. The existing serial console may be connected to the source vm. So when we ping vm on dest, we need to clean the console first. Then we create a new serial console for vm on dest.